packages/async-rewriter/package.json

```diff
@@ -36,10 +36,10 @@
     "@types/babel__core": "^7.1.6",
     "@types/babel__traverse": "^7.0.9",
     "acorn": "^7.2.0",
-    "acorn-class-fields": "^0.3.2",
-    "acorn-numeric-separator": "^0.3.2",
-    "acorn-private-methods": "^0.3.1",
-    "acorn-static-class-features": "^0.2.1",
+    "acorn-class-fields": "^0.3.7",
```

Comment:
do we want to pin those? or is too much paranoid?

Reply:
hah. doesn't hurt. Let's do it. Pushed update